backend/routers/transcribe_v2.py

```diff
@@ -443,14 +447,14 @@ async def stream_transcript_process():
                 # Send to client
                 await websocket.send_json(segments)
 
-                # Send to external trigger
-                if transcript_send:
-                    transcript_send(segments)
-
                 memory = _get_or_create_in_progress_memory(segments)  # can trigger race condition? increase soniox utterance?
                 memories_db.update_memory_segments(uid, memory.id, [s.dict() for s in memory.transcript_segments])
                 memories_db.update_memory_finished_at(uid, memory.id, finished_at)
 
+                # Send to external trigger
```

Comment:
move this code block back to the :446 line please. 

accept none for memory_id in the initial segments (just in case). 

we should not wait for memory updates to complete every time before pushing new segments.

Reply:
But then we won't have access to the memory id, will move it above `memories_db.update_memory_segments` and right below `_get_or_create_in_progress_memory `? Or will try to see if there's another way to access the memory id

Reply:
think more about the reason behind.

>we should not wait for memory updates to complete every time before pushing new segments.

Reply:
Did the suggested change